packages/sanes-chrome-extension/src/routes/requests/index.dom.spec.ts

```diff
@@ -0,0 +1,61 @@
+import TestUtils from 'react-dom/test-utils';
+import { Store } from 'redux';
+import { Request } from '../../extension/background/actions/createPersona/requestHandler';
+import { aNewStore } from '../../store';
+import { RootState } from '../../store/reducers';
+import { whenOnNavigatedToRoute } from '../../utils/test/navigation';
+import { SHARE_IDENTITY, TX_REQUEST } from '../paths';
+import { travelToRequests } from './test/travelToRequests';
+
+describe('DOM > Feature > Requests', () => {
+  const REQUEST_ONE: Request = {
+    id: 1,
+    type: 'getIdentities',
+    reason: 'Reason 1',
+    sender: 'Sender 1',
+    accept: jest.fn(),
+    reject: jest.fn(),
+  };
+
+  const REQUEST_TWO: Request = {
+    id: 2,
+    type: 'signAndPost',
+    reason: 'Reason 2',
+    sender: 'Sender 2',
+    accept: jest.fn(),
+    reject: jest.fn(),
+  };
+
+  let store: Store<RootState>;
+  let requestsDom: React.Component;
+
+  beforeEach(async () => {
+    store = aNewStore();
+  });
+
+  it('redirects to the Share Identity view when a Request of type "getIdentities" is clicked', async () => {
+    requestsDom = await travelToRequests(store, [REQUEST_ONE, REQUEST_TWO]);
+    const identityRequest = TestUtils.scryRenderedDOMComponentsWithTag(requestsDom, 'li')[0];
+
+    TestUtils.act(
+      (): void => {
+        TestUtils.Simulate.click(identityRequest);
+      },
+    );
+
+    await whenOnNavigatedToRoute(store, SHARE_IDENTITY);
+  });
```

Comment:
You need to destroy persona after each test.
```
const persona = getCreatedPersona();
persona.destroy();
```

Otherwise connections are open and probably make the other tests to fail.

Reply:
I have tried that but it does not return any Persona since it's not created anywhere.